CODEOWNERS

```diff
@@ -229,6 +229,7 @@
 /.github/actions/ @cilium/github-sec @cilium/ci-structure
 /.github/workflows/ @cilium/github-sec @cilium/ci-structure
 /.github/workflows/*datapath*.yaml @cilium/sig-datapath @cilium/github-sec @cilium/ci-structure
+/.github/workflows/*ipsec*.yaml @cilium/ipsec @cilium/github-sec @cilium/ci-structure
```

Comment:
Is ci-structure needed here in addition to github-sec? If I look at the diff between the two teams, I'm not seeing anyone familiar with this workflow structure that is in ci-structure but not in github-sec.

Reply:
I didn't want to change it in order to keep it future-proof.